### PR TITLE
updated podmon version in installation wizard

### PR DESCRIPTION
**Pull request by niranjan-n1**
# Description

1. Added operator template for csm 1.12
2. updated podmon and replication version in installation wizard
3. updated observability version in installation wizard
4. Helm chart changes addressed in https://github.com/dell/helm-charts/pull/504

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1435|

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [] Have you tested whether the hyperlinks are working properly?
- [] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by gallacher — DISMISSED** [review]
@niranjan-n1 - please ensure all version updates in this repository are happening at the same time to drive consistency.  Thanks!

**Comment by niranjan-n1** [comment]
> @niranjan-n1 - please ensure all version updates in this repository are happening at the same time to drive consistency. Thanks!

Hi @gallacher, Installation wizard changes for csm 1.12 for drivers in helm is already added for the drivers in this PR https://github.com/dell/csm-docs/pull/1233/files#diff-8c916df10fa4b13e493875bfac6c8ad7658fb3040242b671f34c357c2edf73ce.

File: content/docs/deployment/csminstallationwizard/src/templates/operator/csm-isilon-1.12.0.template [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+            - name: "COLLECTOR_ADDRESS"
+              value: "otel-collector:55680"
+    - name: resiliency
+      # enabled: Enable/Disable Resiliency feature
+      # Allowed values:
+      #   true: enable Resiliency feature(deploy podmon sidecar)
+      #   false: disable Resiliency feature(do not deploy podmon sidecar)
+      # Default value: false
+      enabled: $OPERATOR_RESILIENCY_ENABLED
+      configVersion: v1.10.0
```
[the remark this answers is not in the document]
Comment (niranjan-n1) [reply content/docs/deployment/csminstallationwizard/src/templates/operator/csm-isilon-1.12.0.template]: Addressed

**Review by khareRajshree — APPROVED** [review]

**Review by karthikk92 — APPROVED** [review]

**Review by samihan-dell — APPROVED** [review]

File: content/docs/deployment/csminstallationwizard/src/templates/operator/csm-isilon-1.12.0.template [hunk 2]
```diff
@@ -0,0 +1,484 @@
+#
```
Comment (santhoshatdell) [inline content/docs/deployment/csminstallationwizard/src/templates/operator/csm-isilon-1.12.0.template:1]: Can the CSM 1.8 templates be deleted now?

**Review by shefali-malhotra — APPROVED** [review]

File: content/docs/deployment/csminstallationwizard/src/templates/operator/csm-isilon-1.12.0.template (hunk 2 above)
Comment (niranjan-n1) [reply content/docs/deployment/csminstallationwizard/src/templates/operator/csm-isilon-1.12.0.template:1]: Will check with the component owners and update. Thanks @santhoshatdell

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Pull request merged**